sample-app/src/registerServiceWorker.ts

```diff
@@ -0,0 +1,6 @@
+export default function registerServiceWorker() {
+  window.addEventListener('load', () => {
+    const swUrl = `${process.env.PUBLIC_URL}/service-worker.js`;
```

Comment:
Is `PUBLIC_URL` something `create-react-app` makes available for us automagically?

Reply:
yup! this particular line is taken from the example service worker registration in the PWA (progressive web app) create-react-app template https://github.com/cra-template/pwa/blob/master/packages/cra-template-pwa-typescript/template/src/serviceWorkerRegistration.ts